lib/PuppeteerSharp.Tests/TargetTests/CreateCDPSessionTests.cs

```diff
@@ -72,7 +73,7 @@ public async Task ShouldBeAbleToDetachSession()
             var client = await Page.Target.CreateCDPSessionAsync();
             await client.SendAsync("Runtime.enable");
             var evalResponse = await client.SendAsync("Runtime.evaluate", new { expression = "1 + 2", returnByValue = true });
-            Assert.Equal(3, evalResponse.result.value.ToObject<int>());
+            Assert.Equal(3, evalResponse["result"]["value"].ToObject<int>());
```

Comment:
Just thinking about an example. Would a user be able to do this?
```
dynamic evalResponse = await client.SendAsync<ExpandoObject>("Runtime.evaluate", new { expression = "1 + 2", returnByValue = true });
Assert.Equal(3, evalResponse.result.value.ToObject<int>());
```

Reply:
Even easier;

dynamic evalResponse = await client.SendAsync("Runtime.evaluate", new { expression = "1 + 2", returnByValue = true });
Assert.Equal(3, evalResponse.result.value.ToObject<int>());

Or

var evalResponse = await client.SendAsync("Runtime.evaluate", new { expression = "1 + 2", returnByValue = true });
Assert.Equal(3, evalResponse.SelectToken("result.value"));